resources/lang/ko.js

```diff
@@ -21,47 +21,47 @@ class CactbotLanguageKo extends CactbotLanguage {
       Medicated: '강화약', // tbc
       BattleLitany: '전투 기도', // 0x312
       Embolden: '성원', // 0x4d7
-      Arrow: 'FIXME', // 0x75c
+      Arrow: '오쉬온의 화살', // 0x75c
       Balance: '아제마의 균형', // 0x75a
-      Bole: 'FIXME', // 0x75b
-      Ewer: 'FIXME', // 0x75e
-      Spear: 'FIXME', // 0x75d
-      Spire: 'FIXME', // 0x75f
-      LadyOfCrowns: 'FIXME', // 0x755
-      LordOfCrowns: 'FIXME', // 0x754
+      Bole: '세계수의 줄기', // 0x75b
+      Ewer: '살리아크의 물병', // 0x75e
+      Spear: '할로네의 창', // 0x75d
+      Spire: '비레고의 탑', // 0x75f
+      LadyOfCrowns: '점지: 여왕의 날개', // 0x755
```

Comment:
It looks like this is different from what was added in #887.  Which of these is the correct version?